src/com/owncloud/android/db/PreferenceManager.java

```diff
@@ -299,4 +299,4 @@ public void setSourcePath(String sourcePath) {
             mSourcePath = sourcePath;
         }
     }
-}
+}
```

Comment:
git is complaining because you have removed the latest empty blank line of the file ;-)